src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,62 @@
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { useAppSelector } from '../../app/hooks';
+import { useDispatch } from 'react-redux';
+import { actions } from '../../features/currentTodo';
+
+type Prop = {
```

Comment:
u forgot 1 letter here)
```suggestion
type Props = {
```